genomic_data_service/regulome_atlas.py

```diff
@@ -2,42 +2,80 @@
 import math
 import pyBigWig
 
-RESIDENT_REGIONSET_KEY = 'resident_regionsets'  # keeps track of what datsets are resident
-FOR_REGULOME_DB = 'regulomedb'
-EVIDENCE_CATEGORIES = ['QTL', 'ChIP', 'DNase', 'PWM', 'Footprint', 'PWM_matched', 'Footprint_matched']
+RESIDENT_REGIONSET_KEY = (
+    "resident_regionsets"  # keeps track of what datsets are resident
+)
+FOR_REGULOME_DB = "regulomedb"
+EVIDENCE_CATEGORIES = [
+    "QTL",
+    "ChIP",
+    "DNase",
+    "PWM",
+    "Footprint",
+    "PWM_matched",
+    "Footprint_matched",
+]
 
 # when iterating scored snps or bases, chunk calls to index for efficiency
 # NOTE: failures seen when chunking is too large
 REGDB_SCORE_CHUNK_SIZE = 30000
 
 # RegulomeDB scores for bigWig (bedGraph) are converted to numeric and can be converted back
-REGDB_STR_SCORES = ['1a', '1b', '1c', '1d', '1e', '1f', '2a', '2b', '2c', '3a', '3b', '4', '5', '6']
-REGDB_NUM_SCORES = [1000, 950, 900, 850, 800, 750, 600, 550, 500, 450, 400, 300, 200, 100]
+REGDB_STR_SCORES = [
+    "1a",
+    "1b",
+    "1c",
+    "1d",
+    "1e",
+    "1f",
+    "2a",
+    "2b",
+    "2c",
+    "3a",
+    "3b",
+    "4",
+    "5",
+    "6",
+]
+REGDB_NUM_SCORES = [
+    1000,
+    950,
+    900,
+    850,
+    800,
+    750,
+    600,
+    550,
+    500,
+    450,
+    400,
+    300,
+    200,
+    100,
+]
 
 try:
-    TRAINED_REG_MODEL = pickle.load(
-        open('./ml_models/rf_model1.0.1.sav', 'rb')
-    )
+    TRAINED_REG_MODEL = pickle.load(open("./ml_models/rf_model1.0.1.sav", "rb"))
 except FileNotFoundError:
     TRAINED_REG_MODEL = None
 
 try:
     IC_MATCHED_MAX_BW = pyBigWig.open(
-        './ml_models/bigwig_files/IC_matched_max.bw'
+        "https://regulome-ml-models.s3.amazonaws.com/bigwig_files/IC_matched_max.bw"
```

Comment:
probably we should check if the local file is there before using the online one. files are big enough to warrant that imo.